wlauto/common/android/device.py

```diff
@@ -670,6 +670,18 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_network_connected(self, network_host='www.google.com'):
```

Comment:
My apologies that was as copy and paste error! Should be correct now. Ran the pylint script and that file is clean.